### PR TITLE
Remove unsigned option

### PR DESCRIPTION
**Pull request by c2bo**
Still needs some work. I've kept the overall document structure as it felt cleaner to me this way instead of moving status list data structures directly into the JWT/CWT sections.

Rendered version: https://drafts.oauth.net/draft-ietf-oauth-status-list/c2bo/remove-unsigned-option/draft-ietf-oauth-status-list.html

### CODE REVIEW

File: draft-ietf-oauth-status-list.md [hunk 1]
```diff
@@ -81,15 +81,15 @@ informative:
...
+This document defines a Status List and its representations in JWT and CWT formats that describe the individual statuses of multiple Referenced Tokens, which themselves are JWTs or CWTs. The statuses of all Referenced Tokens are conveyed via a bit array in the Status List. Each Referenced Token is allocated an index during issuance that represents its position within this bit array. The value of the bit(s) at this index correspond to the Referenced Token's status. A Status List is provided within a Status List Token protected by cryptographic signature or MAC and this document defines its representations in JWT and CWT format. Status Lists may be composed for expressing a range of Status Types. This document defines basic Status Types for the most common use cases as well as an extensibility mechanism for custom Status Types. The document also defines how an issuer of a Referenced Token references a Status List Token.
```
Comment (tplooker) [inline draft-ietf-oauth-status-list.md:92]:
```suggestion
This document defines a Status List and its representations in JWT and CWT formats that describe the individual statuses of multiple Referenced Tokens, which themselves are often JWTs or CWTs. The statuses of all Referenced Tokens are conveyed via a bit array in the Status List. Each Referenced Token is allocated an index during issuance that represents its position within this bit array. The value of the bit(s) at this index correspond to the Referenced Token's status. A Status List is provided within a Status List Token protected by cryptographic signature or MAC and this document defines its representations in JWT and CWT format. Status Lists may be composed for expressing a range of Status Types. This document defines basic Status Types for the most common use cases as well as an extensibility mechanism for custom Status Types. The document also defines how an issuer of a Referenced Token references a Status List Token.
```

Not sure if often is sufficient here but probably need to leave room for say mDoc which isn't technically a CWT.

**Comment by tplooker** [comment]
Looking good there is still some text in section 8.4 referring to the unsigned option that will need to be removed too.

File: draft-ietf-oauth-status-list.md [hunk 2]
```diff
@@ -114,7 +115,7 @@ The following diagram depicts the relationship between the involved roles (Relyi
 
 ~~~
 
-An Issuer issues Referenced Tokens to a Holder, the Holder uses and presents those Referenced Tokens to a Relying Party. The Issuer gives updated status information to the Status Issuer, who creates a Status List (Token). The Status Issuer gives the Status List (Token) to the Status Provider, who serves the Status List (Token) on a public, resolvable endpoint. The roles of the Issuer (of the Referenced Token), the Status Issuer and the Status Provider may be fulfilled by the same entity. If not further specified, the term Issuer may refer to an entity acting for all three roles.
+An Issuer issues Referenced Tokens to a Holder, the Holder uses and presents those Referenced Tokens to a Relying Party. The Issuer gives updated status information to the Status Issuer, who creates a Status List Token. The Status Issuer provides the Status List Token to the Status Provider, who serves the Status List (Token) on a public, resolvable endpoint. The roles of the Issuer (of the Referenced Token), the Status Issuer and the Status Provider may be fulfilled by the same entity. If not further specified, the term Issuer may refer to an entity acting for all three roles.
```
Comment (paulbastian) [inline draft-ietf-oauth-status-list.md:118]:
```suggestion
An Issuer issues Referenced Tokens to a Holder, the Holder uses and presents those Referenced Tokens to a Relying Party. The Issuer gives updated status information to the Status Issuer, who creates a Status List Token. The Status Issuer provides the Status List Token to the Status Provider, who serves the Status List Token on a public, resolvable endpoint. The roles of the Issuer (of the Referenced Token), the Status Issuer and the Status Provider may be fulfilled by the same entity. If not further specified, the term Issuer may refer to an entity acting for all three roles.
```

File: draft-ietf-oauth-status-list.md [hunk 3]
```diff
@@ -167,10 +168,10 @@ Issuer:
 : An entity that issues the Referenced Token.
 
 Status Issuer:
-: An entity that issues the Status List or Status List Token about the status information of the Referenced Token. This role may be fulfilled by the Issuer.
+: An entity that issues the Status List Token about the status information of the Referenced Token. This role may be fulfilled by the Issuer.
 
 Status Provider:
-: An entity that provides the Status List or Status List Token on a public endpoint. This role may be fulfilled by the Status Issuer.
+: An entity that provides the Status List Token on a public endpoint. This role may be fulfilled by the Status Issuer.
```
Comment (paulbastian) [inline draft-ietf-oauth-status-list.md:174]: Relying Party: Status List -> Status List Token

**Review by paulbastian — APPROVED** [review]

**Review by tplooker — APPROVED** [review]

**Pull request merged**